latch/resources/tasks.py

```diff
@@ -47,6 +47,35 @@ def my_task(a: int) -> str:
 from .dynamic import DynamicTaskConfig
 
 
+def _get_giant_gpu_pod() -> Pod:
```

Comment:
lol can we convene on just naming what it is "_get_v100_4x_pod()"

Reply:
was doing it for the meme lol